.changeset/rich-moons-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-documentation': patch
+---
+
+Static assets for the docs can now be loaded from [xy] folder and @swisspost/design-system-components can now be used across the whole documentation.
```

Comment:
Replace xy by whatever the new static folder is. But I think this info is irrelevant to devs outside the contributor circle. Therefore I think this changeset can actually be deleted.